### PR TITLE
Incrase memory for app controller

### PR DESCRIPTION
**Pull request by HumairAK**
Application controller memory increase

### CODE REVIEW

File: objects/namespace-install.yaml [hunk 1]
```diff
@@ -2255,7 +2255,7 @@ spec:
         name: argocd-application-controller
         resources:
           limits:
-            memory: 2Gi
```
Comment (anishasthana) [inline objects/namespace-install.yaml:2258]: can you add a memory request for 4Gi as well?

Comment (anishasthana) [reply objects/namespace-install.yaml:2258]: the commit message also says `Incrase`

Comment (HumairAK) [reply objects/namespace-install.yaml:2258]: request defaults to limits if unspecified and limit is explicitly specified

Comment (anishasthana) [reply objects/namespace-install.yaml:2258]: We've had this conversation before haven't we... Just fix the commit message :-)

**Comment by HumairAK** [comment]
fixed commit message

**Pull request merged**